fili-core/src/main/java/com/yahoo/bard/webservice/web/filters/QueryParameterNormalizationFilter.java

```diff
@@ -73,16 +74,18 @@ public void filter(ContainerRequestContext containerRequestContext) throws IOExc
      * @return  Normalized URI
      */
     private URI buildNormalizedUri(UriInfo uriInfo) {
-        UriBuilder builder = uriInfo.getRequestUriBuilder();
-
-        // Erase existing query parameters from builder
-        builder.replaceQuery("");
+        // Get the URI without any query params
+        URIBuilder builder = new URIBuilder(uriInfo.getAbsolutePath());
 
         // Set normalized values
         getNormalizedQueryParameters(uriInfo)
-                .forEach(keyValue -> builder.queryParam(keyValue.getLeft(), keyValue.getRight()));
+                .forEach(keyValue -> builder.addParameter(keyValue.getLeft(), keyValue.getRight()));
 
-        return builder.build();
+        try {
+            return builder.build();
+        } catch (URISyntaxException e) {
+            throw new RuntimeException("failed to build uri", e);
```

Comment:
We should log the exception here (probably at the `error` level), otherwise there's a risk we won't log the exception, or at least no the way we do everywhere else. (I don't think errors in filters have a top-level catch wrapped around them the same way that other places do)

We may even want to go so far as to kill the request by responding with a 500 error here, rather than bubbling the exception and having Jersey deal with the response, since it gives us more control over the response payload / message, etc.